cv32e40s/regress/cv32e40s_full.yaml

```diff
@@ -386,12 +386,6 @@ tests:
     dir: cv32e40s/sim/uvmt
     cmd: make test TEST=debug_test_reset
 
-  debug_test_trigger:
-    description: Debug trigger test
-    builds: [ uvmt_cv32e40s, uvmt_cv32e40s_clic]
-    dir: cv32e40s/sim/uvmt
-    cmd: make test TEST=debug_test_trigger
```

Comment:
Note: This was overwritten further down.

Reply:
What do you mean overwritten further down? Does it mean that this test was not run? And that it only run with the uvmt_cv32e40s_debug_trigger_cfgs, and not uvmt_cv32e40s_clic? I am not really understanding it...

Reply:
> What do you mean overwritten further down?

Further down, in the same file, that test was defined twice.

The second definition takes precedence, so the first one is superfluous (and also erronous).

![bilde](https://github.com/openhwgroup/core-v-verif/assets/75431602/36c06854-5661-4052-b213-33415bde3256)

Reply:
> And that it only run with the uvmt_cv32e40s_debug_trigger_cfgs, and not uvmt_cv32e40s_clic? I am not really understanding it...

I found the problem like this:
I wanted to run the test, so I confered the "full" yaml to see what config to run with.
Then it failed.
It can seemingly not run with those configs, it needs those configs who are in the second definition.